doc/central_scheduler.rst

```diff
@@ -27,7 +27,7 @@ To run the server as a daemon run:
 
 .. code-block:: console
 
-    $ luigid --background --pidfile <PATH_TO_PIDFILE> --logdir <PATH_TO_LOGDIR> --state-path <PATH_TO_STATEFILE>
+    $ luigid --background --pidfile <PATH_TO_PIDFILE> --logdir <PATH_TO_LOGDIR> --state_path <PATH_TO_STATEFILE>
```

Comment:
are you sure about this one?? I think  it should be `-` as that's what use for all other command line parameters

Reply:
Yes, my bad. There should be -.